contributing/telemetry.md

```diff
@@ -4,9 +4,12 @@ Similar to what vscode does, we are reporting on certain events that happen in t
 
 ## How it works
 
-All extensions leverage the telemetryService in `salesforce-vscode-core` extension in order to send data. The service in `salesforce-vscode-core` is responsible for showing an information message to users and properly initializing the service for other extensions in this repository to leverage. Similar to VSCode, we provide the user an opt out mechanism. We check that user settings `telemetry.enableTelemetry` and `salesforcedx-vscode-core.telemetry.enabled` are enabled before initializing the service.
+All extensions leverage the telemetryService in `salesforce-vscode-core` extension in order to send data. 
 
-Additionally, if telemetry is disabled for Salesforce CLI using the `disable-telemetry` configuration setting or the `SF_DISABLE_TELEMETRY` environment variable, the telemetryService doesn't send data.
+There are several ways to disable telemetry for all Salesforce extensions.
+- Turn off the Core Extension telemetry setting at the workspace `("salesforcedx-vscode-core.telemetry.enabled": false)`
+- Disable SF CLI telemetry 
+  - `sf config set disable-telemetry=true --global`
```

Comment:
Added specific list of how to disable telemetry

Reply:
we have not called out internal users use case here. is that intentional?

Reply:
wonder why we have 2 setting instead of just 1?

Reply:
heh...we really have 3, but the top level vscode setting is no longer valid since VSCode made a change to how to manage this (when did this change happen? no idea... I'm pretty sure before I was ever working on this product so at least 3 years ago).

Reply:
I view these docs as general and the internal use case is specific to just SFDC users.  Think we should call it out here?

Reply:
Unless you see any downside to it, it will be good to capture it here so we know what to expect for internal users?